libs/java/server_common/src/main/resources/messages/ServerCommon.properties

```diff
@@ -10,3 +10,9 @@ athenz.notification.email.principal.expiry.body.entry=<tr><td class="cv">{0}</td
 
 athenz.notification.email.unrefreshed.certs.subject=Athenz Unrefreshed Certificates Notification
 athenz.notification.email.unrefreshed.certs.body.entry=<tr><td class="cv">{0}</td><td class="cv">{1}</td><td class="cv">{2}</td><td class="cv">{3}</td><td class="cv">{4}</td><td class="cv">{5}</td></tr>
```

Comment:
have left ServerCommon.properties in common even tough this file contains notification specific attributes.
I will move these attributes to files in zms / zts on a separate pull request.